charts/orchestrator/README.md

```diff
@@ -75,6 +75,18 @@ The following table lists the configurable parameters of the Orchestrator chart
 | `tekton.enabled` | whether to create the Tekton pipeline resources | `false` |
 | `argocd.enabled` | whether to install the ArgoCD plugin and create the orchestrator AppProject | `false` |
 | `argocd.namespace` | Defines the namespace where the orchestrator's instance of ArgoCD is deployed. The value is captured when running setup.sh script and stored as a label in the selected namespace. User can override the value by populating this field. Defaults to `orchestrator-gitops` in the setup.sh script. | `""` |
+| `rhsso.enabled` | Whether to create the Red Hat SSO login information | `true` |
+| `rhsso.deploy` | If we need to deploy the RedHat SSO operator (For development pourposes) | `true` |
```

Comment:
deploy is when you want to add keycloak resources, enable when you only want to consume it.

Reply:
understood. Should the chart check if there is an existing installation when `deploy==false` and `enabled==true`?

Reply:
can be keycloak or any other OIDC integration, can be an external thing.

Reply:
but is there a way to know via chart information what to expect? I would be good if we could add this into the templating logic so that we prevent the deployment of the chart/CR instance in an environment that is not properly configured. If not possible then it's fine... it's just a nice thing to have.